crates/nu-command/src/filesystem/save.rs

```diff
@@ -318,10 +322,20 @@ fn stream_to_file(
     mut stream: RawStream,
     file: File,
     span: Span,
+    progress: bool,
 ) -> Result<PipelineData, ShellError> {
     let mut writer = BufWriter::new(file);
 
-    stream
+    let mut bytes_processed: u64 = 0;
+    let bytes_processed_p = &mut bytes_processed;
+    let file_total_size = stream.known_size.map(|n| (n as u64)); // type TBD
```

Comment:
It would be nice to avoid this `map` call. Maybe `known_size` should be `Option<u64>` instead of `Option<usize>`?